src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionDocumentEntity.java

```diff
@@ -97,4 +97,11 @@ public class TranscriptionDocumentEntity extends ModifiedBaseEntity
 
     @OneToMany(mappedBy = ObjectAdminActionEntity_.TRANSCRIPTION_DOCUMENT)
     private List<ObjectAdminActionEntity> adminActions = new ArrayList<>();
+
+    @Column(name = "subcontent_object_id")
+    private String subcontentObjectId;
+
+    @Column(name = "subcontent_position")
+    private Integer subcontentPosition;
```

Comment:
Validation (See previous comment)?

Reply:
See earlier response to similar comment.